YMChat/src/main/assets/index.html

```diff
@@ -0,0 +1,152 @@
+
+
+        let payloadJSON = {};
+        try{
+            payloadJSON = JSON.parse(payload);
+        }catch(e){
+        }
+
+
+        window.ymConfig = {
+            bot: botId || '',
+            pwa: true,
+            sdk: true, // Currently only used for analytics
+            enableHistory: (enableHistory === 'true') ,
+            ymAuthenticationToken: ymAuthenticationToken,
+            deviceToken: deviceToken,
+            logging: true,
+            handlers: {
+                sendIOSEvent: function (data) {
+                 console.log(data);
+
+                    try {
+                        window.webkit.messageHandlers.ymHandler.postMessage(data);
+                    } catch (e) { }
+                    try {
+                        window.YMHandler.receiveMessage(JSON.stringify(data));
+                    } catch (e) { }
+                    try {
+                        window.ReactNativeWebView.postMessage(JSON.stringify(data));
+                    } catch (e) { }
+
+                }
+            },
+            payload: payload,
+            triggerJourneyOnEnableHistory: payloadJSON.JourneySlug ? true: false ,
+            triggerJourney: payloadJSON.JourneySlug
+        };
+
+        if(payloadJSON.UserId) {
+            window.ymConfig.userId = payloadJSON.UserId;
+        }
+
+        function sendEvent(s = "") {
+            const iframe = document.getElementById('ymIframe');
+            iframe.contentWindow.postMessage(JSON.stringify({
+                    event_code: 'send-voice-text',
+                    data: s
+                }));
+
+
+            try {
+                window.YMHandler.receiveMessage(JSON.stringify({ code: "Message Received", data: s }));
+            } catch (e) { }
+            try {
+                window.webkit.messageHandlers.ymHandler.postMessage({ code: "Message Received", data: s });
+            } catch (e) { }
+        }
+
+        (function () {
+            var w = window, ic = w.YellowMessenger;
+            if ("function" === typeof ic)
+                ic("reattach_activator"), ic("update", ymConfig);
+            else {
+                var d = document, i = function () {
+                    i.c(arguments)
+                };
+
+                function l() {
+                    var e = d.createElement("script");
+                    e.type = "text/javascript", e.async = !0, e.src = (customBaseUrl)+"/widget/mobile.js";
+                    var t = d.getElementsByTagName("script")[0];
+                    t.parentNode.insertBefore(e, t)
+                }
+               /*
+                function l() {
+                    var e = d.createElement("script");
+                    e.type = "text/javascript", e.async = !0, e.src = "https://botplatform.io/components/securedwidget/mobile.js";
+                    var t = d.getElementsByTagName("script")[0];
+                    t.parentNode.insertBefore(e, t)
+                }
+                */
```

Comment:
Do we need this?

Reply:
Let me have confirmation for this. 
As `https://botplatform.io/components/securedwidget/mobile.js` was created for security reasons and whitelisted domains. we can either use `https://botplatform.io/components/securedwidget/mobile.js` or `https://app.yellowmessenger.com`